README.md

```diff
@@ -68,9 +68,18 @@ webdriver-manager update      // Downloads the latest binaries.
 webdriver-manager start       // Starts the selenium server standalone.
 ```
 
-Note: Installing globally will not work with Protractor if you are trying to
+Notes:
+* Installing globally will not work with Protractor if you are trying to
 start a Selenium Standalone server with a "local" or "directConnect", because
 Protractor installs its own version of webdriver-manager as a dependency.
+* Right now there is an issue with downloading the latest ChromeDriver.
+It might not be compatible with the current version of Chrome.
+The latest available version of ChromeDriver is the *v74.x.x.x*
+but this one only works with Chrome *v74.x.x.x* which is in Beta right now.
+So, if you have installed the latest stable version of Chrome (*v73.x.x.x*)
+you should pin that ChromeDriver version to be
+downloaded: `webdriver-manager update --versions.chrome=<73.0.3683.68>`.
```

Comment:
Removing the `<` and `>` will make it easier to copy and paste.
`webdriver-manager update --versions.chrome=73.0.3683.68`

Reply:
fixed